src/screens/FeedbackScreen.tsx

```diff
@@ -0,0 +1,217 @@
+import { screen_names } from '../constants/ScreenNames';
+import { ParamListBase } from '@react-navigation/native';
+import { fonts } from '../assets/fonts/fonts';
+import { colors } from '../constants/Colors';
+
+const FeedBackScreen = ({navigation}: NativeStackScreenProps<ParamListBase, screen_names.FEEDBACK, undefined>) : JSX.Element => {
+    const [modalVisible, setModalVisible] = useState(false);
+    const [text, onChangeText] = React.useState("");
+    const [selection, setSelection] = useState("");
+    const GreatEmoji = { emoji : "😊" , text : "Great" }
+    const OkayEmoji = { emoji : "😐" , text : "Okay" }
+    const BadEmoji =  { emoji : "😔" , text : "Bad"}
+    return (
+      <ScrollView>
+        <View style={styles.main_container}>
+            <Text style={styles.feedback_text}>Your feedback helps us improve</Text>
+            <View style={styles.response_container}>
+                <Text style={styles.response_container_text}>
+                How is/was the event
+                </Text>
+                <View style={styles.emojis_response_container}>
+                  <TouchableOpacity onPress={() => setSelection(BadEmoji.text)} >
+                    <Emojis emojitype={BadEmoji} containerstyle={ selection == BadEmoji.text ? [styles.emoji_container,styles.clicked_emoji_container] : styles.emoji_container}/>
+                  </TouchableOpacity>
+                  <TouchableOpacity onPress={() => setSelection(OkayEmoji.text)} >
+                    <Emojis emojitype={OkayEmoji} containerstyle={ selection == OkayEmoji.text ? [styles.emoji_container,styles.clicked_emoji_container] : styles.emoji_container}/>
+                  </TouchableOpacity>
+                  <TouchableOpacity onPress={() => setSelection(GreatEmoji.text)} >
+                    <Emojis emojitype={GreatEmoji} containerstyle={ selection == GreatEmoji.text ? [styles.emoji_container,styles.clicked_emoji_container] : styles.emoji_container}/>
+                  </TouchableOpacity>
+                </View>
+            </View>
+            <TextInput returnKeyType='done'  onSubmitEditing={() => {Keyboard.dismiss()}} style={styles.textinput_text} value={text} onChangeText={onChangeText} multiline={true} placeholder={"Type message here"}></TextInput>
+            <Modal
+             transparent
+             statusBarTranslucent={false}
+             visible={modalVisible}
+             onRequestClose={() => {
+          setModalVisible(!modalVisible);
+        }}
+      >
+          <View style={styles.modal_view}>
+            <Image source={require("../assets/confetti.png")} style={styles.modal_image}/>
+            <Text style={styles.modal_text}>Thank you for your feedback</Text>
+            <Pressable
+              style={styles.modal_button}
+              onPress={() => navigation.navigate(screen_names.HOME)}
+            >
+              <Text style={styles.modal_button_text}>OKAY</Text>
+            </Pressable>
+          </View>
+      </Modal>
```

Comment:
The modal is actually okay design-wise
The few tweaks needed are :
- Add the dark background overlay like the [design](https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/1de29b30-29d7-4847-9a5d-90a60ff44249/specs/). You can check out HomeScreenNotLoggedIn on how you can do that.
- Match the modal width to the one on the [design](https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/1de29b30-29d7-4847-9a5d-90a60ff44249/specs/). You can also check out HomeScreenNotLoggedIn for ideas.